src/components/filter/filter.tsx

```diff
@@ -120,7 +120,7 @@ export class Filter implements InteractiveComponent {
   //--------------------------------------------------------------------------
 
   componentWillLoad(): void {
-    this.filter(this.value);
+    this.updateFiltered(filter(this.items, this.value));
```

Comment:
This is so that creating the filter initially is not debounced. Allows the `filteredItems` to be defined initially.